lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,103 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch, dry_run: true)
+        @branch  = branch || 'master'
+        @dry_run = dry_run
+      end
+
+      def execute(command, description = nil)
+        description ||= "*** Running `#{command}` ***"
+        puts description
+        system(command)
+      end
+
+      def create_database_yml
+        puts "*** Creating config/database.yml ***"
+        database_config = YAML.load File.read("config/database.pg.yml")
+        unless database_config.key?('i18n')
+          database_config['i18n'] = database_config['test'].dup
+          database_config['i18n']['database'] = "vmdb_i18n"
+        end
+        File.open("config/database.yml", "w") { |file| file.write(database_config.to_yaml) }
```

Comment:
```suggestion
        File.write("config/database.yml", database_config.to_yaml)
```

Reply:
Also, no need to write if you don't change anything, so probably better to have this line inside the conditional